ws-nextjs-app/pages/[service]/send/[id]/SuccessMessage/index.tsx

```diff
@@ -0,0 +1,95 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import Heading from '#app/components/Heading';
+import Paragraph from '#app/components/Paragraph';
+import { useContext } from 'react';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import styles from './index.styles';
+import TickSvg from './svgs';
+import { useFormContext } from '../FormContext';
+
+const DEFAULT_RETENTION_POLICY_DAY = '270';
+const DEFAULT_EMAIL = 'CannotFindEmail@bbc.co.uk';
+
+const defaultTranslations = {
+  confirmationStepTitle: 'Message sent',
+  confirmationStepDescriptionHtml: 'Thanks for getting in touch.',
+  referenceNumber: 'Reference number',
+  submissionInfoSignedOutMessage:
+    'You may wish to make a note of these details for your reference.',
+  retentionPeriodDays:
+    "We'll keep your submission for up to {{days}} days – and if we don't use it we'll then delete it and any other information you sent us.",
+  privacyInfoHtml:
+    "Don't worry, we protect your information — read the {{privacyInfoLink}} for more details.",
+  emailToHtml:
+    "If you change your mind and don't want us to use it, just email us at {{emailLink}}. Don't forget the reference number.",
+  removalGuidelineText:
+    'If you submitted something for a programme or online, we won’t be able to remove it once we use it.',
+  privacyPolicyLinkHref: 'https://www.bbc.com/privacy/',
+  privacyPolicyLinkText: 'Privacy Policy',
+};
+
+const SuccessMessage = () => {
+  const {
+    translations: { ugc = defaultTranslations },
+  } = useContext(ServiceContext);
+
+  const { submissionID } = useFormContext();
+
+  const {
+    confirmationStepTitle,
+    confirmationStepDescriptionHtml,
+    submissionInfoSignedOutMessage,
+    referenceNumber,
+    retentionPeriodDays,
+    privacyInfoHtml,
+    emailToHtml,
+    removalGuidelineText,
+    privacyPolicyLinkHref,
+    privacyPolicyLinkText,
+  } = ugc;
+
+  const retentionPolicy = retentionPeriodDays.replace(
+    '{{days}}',
+    DEFAULT_RETENTION_POLICY_DAY,
+  );
+
+  const privacyClauses = privacyInfoHtml.split('{{privacyInfoLink}}');
+  const emailGuidelineClauses = emailToHtml.split('{{emailLink}}');
+
+  return (
+    <div css={styles.outerContainer}>
+      <div css={styles.messageContainer}>
+        <TickSvg css={styles.tickIcon} />
+        <div css={styles.messageTextContainer}>
+          <div>
+            <Heading level={1} id="content" tabIndex={-1} size="trafalgar">
+              {confirmationStepTitle}
+            </Heading>
+            <Paragraph>{confirmationStepDescriptionHtml}</Paragraph>
+          </div>
+        </div>
+      </div>
+      <div css={styles.descriptionContainer}>
+        <Paragraph>{submissionInfoSignedOutMessage}</Paragraph>
+        <div>
+          <Paragraph fontVariant="sansBold">{referenceNumber}</Paragraph>
```

Comment:
Looking at the screen reader UX, I think this should be a strong tag: https://paper.dropbox.com/doc/WS-Uploader-MVP-Screenreader-UX--CP8ef0FMHSpu4ygmsl5KwliLAg-tgOv3vS52cfR5tpaqmJiI#:uid=543995370330451648777476&h2=Form-submitted. I think we can use our Text component for this.